code/game/objects/structures/barsign.dm

```diff
@@ -413,6 +413,17 @@ var/list/barsigns = list()
 	pixel_x = 0
 	pixel_y = 0
 
+//You get actual annoying sounds If its emag'd
+/obj/structure/sign/double/barsign/emag_act(mob/user)
+	sound_selection["Rooster"] = 'sound/misc/6amRooster.wav'
```

Comment:
This actually works?

Reply:
Afaik, an assc list won't make multiple keys of the same thing. emag_act() is also called on the emag afterattack